src/AutoRest.CSharp/LowLevel/Generation/DpgClientWriter.cs

```diff
@@ -292,10 +292,11 @@ private void WriteSubClientInternalConstructor(ConstructorSignature signature)
 
         private void WriteConvenienceMethod(LowLevelClientMethod clientMethod, ConvenienceMethod convenienceMethod, ClientFields fields, bool async)
         {
+            bool isMultipartOperation = convenienceMethod.RequestMediaTypes != null && convenienceMethod.RequestMediaTypes.Count == 1 && convenienceMethod.RequestMediaTypes.Contains("multipart/form-data");
             using (WriteConvenienceMethodDeclaration(_writer, convenienceMethod, fields, async))
             {
                 // This method now builds statements for normal operation and lro operations
-                MethodBodyStatement statement = convenienceMethod.GetConvertStatements(clientMethod, async, fields.ClientDiagnosticsProperty).ToArray();
+                MethodBodyStatement statement = convenienceMethod.GetConvertStatements(clientMethod, async, fields.ClientDiagnosticsProperty, isMultipartOperation).ToArray();
```

Comment:
Shall we move the logic of determining `isMultipartOperation` inside `convenienceMethod.GetConvertStatements()`? `isMultpartOperation` is all determined from properties of `convenienceMethod` itself. So maybe we don't need to calculate it outside `convenienceMethod`.